docker/cassandra/Dockerfile

```diff
@@ -3,3 +3,8 @@ FROM cassandra:3.11
 RUN  apt-get update && apt-get -qq -y install \
        libcap2-bin \
      && setcap cap_ipc_lock=ep $(readlink -f $(which java))
+
+#Make sure we don't use image entrypoint (the bootstrap is done by CassKop)
+ENTRYPOINT []
+#Just start Cassandra
```

Comment:
In order to avoid the default image to configure /etc/cassandra when starting (and because I've already done it in the bootstrap phase, I disable it here)
@cscetbon It would be better to disable this in the Operator no ?

Reply:
Yes Tests e2e where not passing because of entrypoint enabled by default in the official wassandra image